### ISSUE TITLE
rest api to execute individual apps as standalone

### ISSUE THREAD

**Issue by aoighost** [nsacyber/WALKOFF#227]
A REST api to execute individual apps as standalone would help greatly with the ability to execute apps from outside of walkoff individual from workflows

**Comment by adpham95** [comment]
Yes, this would be very useful - it's been on our backlog for a while. We're also planning some webhook features that might tie into this. 

It shouldn't be too big of a lift - I'm currently working on a "bootloader" container to make setup a bit easier, and my next task will be to build out an API for app management on the Umpire container (which manages scaling of apps and workers). Either that, or a routine in the Umpire listening on a Redis stream fed by the REST API on the API Gateway, details TBD. 

That is likely where this feature would end up.
